From 6b60b90b0cfa8e881324533082fbefb6aaaa2b8d Mon Sep 17 00:00:00 2001
From: Markus Koch <markus@notsyncing.net>
Date: Fri, 26 Jul 2019 18:26:30 +0200
Subject: [PATCH 1/2] inputattach: add 155200 to the list of valid baud rates

There are multiple supported devices which work with this baud rate.

Signed-off-by: Markus Koch <markus@notsyncing.net>
---
 utils/inputattach.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/inputattach.c b/utils/inputattach.c
index cdbec739..31cc431f 100644
--- a/utils/inputattach.c
+++ b/utils/inputattach.c
@@ -1010,6 +1010,7 @@ int main(int argc, char **argv)
 	case 9600: type->speed = B9600; break;
 	case 19200: type->speed = B19200; break;
 	case 38400: type->speed = B38400; break;
+	case 115200: type->speed = B115200; break;
 	default:
 		fprintf(stderr, "inputattach: invalid baud rate '%d'\n",
 				baud);
-- 
2.22.0

